buildstockbatch/cloud/docker_base.py

```diff
@@ -160,6 +157,9 @@ def _run_batch_prep(self, tmppath):
         # Collect simulations to queue
         batch_info = self._prep_jobs_for_batch(tmppath)
```

Comment:
Does _prep_jobs_for_batch also consider precomputed sampling? like if the buildstock.csv exists, does this still generate the sampling file?

Reply:
Mainly, I am trying to understand if I need simulation only for couple of buildings that I have generated manually and I don't need to run sampling, how would that be handled here? can you point me to part of the code that explains that part?

Reply:
> like if the buildstock.csv exists, does this still generate the sampling file?

Yes, it ignores the existing file.

> can you point me to part of the code that explains that part?

https://github.com/rewiringamerica/buildstockbatch/blob/fd3a7a9e23833d3371ba05027ff7b7758493d0c2/buildstockbatch/sampler/residential_quota.py#L55